src/components/home/HomePage.js

```diff
@@ -1,41 +1,71 @@
-import React, { Component } from 'react';
+import React, { useState, useEffect } from 'react';
 import './HomePage.css';
 
-class HomePage extends Component {
-  render() {
-    return (
-      <div className="HomePage">
-        <div className="uk-grid">
-          <div className="uk-width-1-5@s" style={{ background: '#0e0b0bb3', color: 'white' }}>
-            <div className="uk-offcanvas-content">
-              <button
-                className="uk-button uk-button-default uk-margin-small-right"
-                type="button"
-                data-uk-toggle="target: #offcanvas-push"
-              >
-                Push
-              </button>
+function HomePage() {
+  const [data, setData] = useState(null);
 
-              <div id="offcanvas-push" data-uk-offcanvas="mode: push; overlay: true">
-                <div className="uk-offcanvas-bar">
-                  <button className="uk-offcanvas-close" type="button" data-uk-close="true"></button>
+  useEffect(() => {
+    fetch('https://cors-anywhere.herokuapp.com/https://api.github.com/repos/iiitv/algos/contents', {
```

Comment:
Can't be called without CORS?

Reply:
No sir I tried it many times..but cors error was there..Even stackoverflow solns were also giving error...only this soln helps me out...so i used this heroku link

Reply:
@mohitkyadav can help with this, he created v1 of algos-web, so definitely worked with this API.

Reply:
take a look at https://github.com/mohitkyadav/algos-ngular/blob/master/src/app/services/content.service.ts#L14
@prathmeshkhandelwal1

Reply:
Sir i fetched the same url & it was giving cors error.

Reply:
@mohitkyadav Can you provide any soln?

Reply:
@aashutoshrathi Sir can you help in this issue?

Reply:
I'll look into the code in a while.
cc: @mohitkyadav

Reply:
okkk Sir